### ISSUE TITLE
MD063 (heading capitalization) does not consider inline code starting a heading

### ISSUE THREAD

**Issue by chrisgrieser** [rvben/rumdl#228]
given this config:

```toml
[MD063]  # heading-capitalization
enabled = true
style = "sentence_case"
```

`rumdl` ignores the first word if it is inline code, resulting in capitalizing "Is", even though it is the second word and should thus be lowercased.

```md
<!-- rumdl expects -->
# `rumdle` Is a linter

<!-- the correct casing is however -->
# `rumdle` is a linter
```

**Comment by rvben** [comment]
Fixed in v0.0.201. Headings starting with inline code now correctly apply sentence case to subsequent text. Thanks @chrisgrieser for reporting!
